### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by anton25360**
Made the landing page, comlpete with sidebar & carousel

### CODE REVIEW

File: app/css/style.css [hunk 1]
```diff
@@ -1,2 +1,92 @@
...
+
+#getRecipeBtn {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  margin-bottom: 25px;
+  padding: 5px 0;
+  background-color: #3ae8a5;
+  color: #304352;
+  z-index: 100; }
```
Comment (harrycoulton) [inline app/css/style.css]: Can see 5, 10, and 100 on the z-index. 

should this follow a more regular pattern, ideally 100, 200, 300 respectively?

File: app/index.php [hunk 2]
```diff
@@ -2,11 +2,127 @@
...
+</nav>
+<div class="nav-transform"></div>
+<div class="container-fluid">
+    <div class="row" id="mainContent">
+        <aside class="col-2 justify-content-center">
+            <form>
+                <input class="choiceBtn col-2" id="getRecipeBtn" type="submit" value="Get Recipe">
+                <input class="choiceBtn" id="expandAll" type="submit" data-toggle="collapse"
+                       data-target=".multi-collapse" aria-expanded="false"
+                       aria-controls="multiCollapseExample1 multiCollapseExample2" value="Expand All">
```
Comment (harrycoulton) [inline app/index.php:36]: Expand all doesn't work? Sorry... :(

Comment (harrycoulton) [reply app/index.php:36]: Checked on both firefox and chrome

Comment (harrycoulton) [reply app/index.php:36]: This may be due to the fact that it's type="submit", as it appears to be refreshing the page each time we click on it.

**Review by harrycoulton — CHANGES_REQUESTED** [review]
Overall good - two issues that need looking at

File: app/index.php (hunk 2 above)
Comment (harrycoulton) [reply app/index.php:36]: One test that we did was that we are able to remove type="submit", which appears to fix the issue. 

You can give it response to the user by giving it a cursor-type on the button, which will achieve the same effect as before without the bug.

Comment (anton25360) [reply app/index.php:36]: removed input type="submit"

File: app/css/style.css (hunk 1 above)
Comment (anton25360) [reply app/css/style.css]: renamed z-index instances

**Pull request merged**